hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -202,6 +207,54 @@ spec = parallel $ do
     propBelowSizeLimit maxTxSize forAllFanout
     propIsValid forAllFanout
 
+  describe "acceptance" $ do
+    it "can close & fanout every collected head" $ do
+      prop_canCloseFanoutEveryCollect
+
+-- * Properties
+
+prop_canCloseFanoutEveryCollect :: Property
+prop_canCloseFanoutEveryCollect = monadicST $ do
```

Comment:
This property might have been nicer, or more natural, to be expressed in the `ModelSpec`, which of course would have required `ModelSpec` to actually run tx validation in the mock chain.

Reply:
I had that thought as well, but am struggling abut with observability and good counterexamples there (I didn't try long).